src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -89,26 +39,95 @@ EOF
             CEPH_PY_VERSION_MAJOR=3
         fi
     fi
+}
 
+setup_teuthology() {
```

Comment:
I tried to install teuthology from pip via setuptools and faced an issue with mysqlclient library, so I've opened a PR there https://github.com/ceph/teuthology/pull/1303.

After that, we may simply install teuthology from github:
`pip install git+https://github.com/ceph/teuthology.git`

And if we remove that part (whith is 33% of this file) why not rewriting this as a tox file?

Reply:
I you should have run bootstrap script before trying to install teuthology and install system packages.